fedimint-cli/src/lib.rs

```diff
@@ -642,22 +659,15 @@ impl FedimintCli {
                 Ok(CliOutput::UntypedApiOutput { value: response })
             }
             Command::Dev(DevCmd::InviteCode) => {
-                let path = cli.workdir()?.join("invite-code");
-                let string = fs::read_to_string(path).map_err_cli_msg(
-                    CliErrorKind::GeneralFederationError,
-                    "cannot read invite code",
+                let invite_code = self.invite_code.clone().ok_or_cli_msg(
```

Comment:
With this change, `fedimint-cli invite-code` only shows the invite code that was previously supplied via `fedimint-cli join-federation`. We need some other way of knowing fed invite code prior to cli initialization.

Reply:
I mean, that's how it would typically go: you get the invite code from someone else already in the federation. For testing I typically just do `cat target/devimint/cfg/invite-code`.